en/battler-tags.json

```
@@ -93,5 +93,7 @@
   "uproarOnCureSleep": "The uproar woke {{pokemonNameWithAffix}}!",
   "uproarOnRemove": "{{pokemonNameWithAffix}} calmed down.",
   "uproarOnPreventSleep": "But the uproar kept {{pokemonNameWithAffix}} awake!",
+  "magicCoatOnAdd": "{{pokemonNameWithAffix}} shrouded itself with Magic Coat!",
+  "magicCoatOnApply": "{{pokemonNameWithAffix}}'s {{moveName}} was bounced back by Magic Coat!",
```

should there be a line break to match the usual text width?
```suggestion
  "magicCoatOnApply": "{{pokemonNameWithAffix}}'s {{moveName}}\nwas bounced back by Magic Coat!",
```

---

I think the line breaks are an anti-pattern in most cases; they lead to inconsistent text widths /shrug

---

Then feel free to merge